shell/platform/windows/egl/egl.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_EGL_EGL_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_EGL_EGL_H_
+
+#include <string>
+
+namespace flutter {
+namespace egl {
+
+/// Log the last EGL error with an error message.
+void LogEGLError(std::string message);
+
+/// Log the last EGL error.
+void LogEGLError(const char* file, int line);
```

Comment:
Similarly consider `(std::string_view file, int line)` here too since it handles both `const char*` and `std::string` params gracefully.

Reply:
Updated to use `std::string_view`